pkg/common/path.go

```diff
@@ -72,14 +82,63 @@ func ExpandPackageDir(f genericclioptions.FileNameFlags) (genericclioptions.File
 		return f, fmt.Errorf("expand package directory should pass one package directory. "+
 			"Passed the following paths: %v", f.Filenames)
 	}
-	configFilepaths, err := expandDir((*f.Filenames)[0])
+	var configFilepaths []string
+	var err error
+	arg := (*f.Filenames)[0]
+	if arg == stdinDash {
+		configFilepaths, err = expandFile(os.Stdin, tmpDir)
+	} else {
+		configFilepaths, err = expandDir(arg)
+	}
 	if err != nil {
 		return f, err
 	}
 	f.Filenames = &configFilepaths
 	return f, nil
 }
 
+// expandFile reads config object yaml from the passed "in" Reader, writing
+// out the config to a newly created temporary file if the config object
+// is not an inventory object. Config objects are separated by the
+// standard "---" separator. Returns the created temp file name in
+// a slice of strings, or an error if one occurred.
+func expandFile(in io.Reader, dir string) ([]string, error) {
+	configFilenames := []string{}
+	// Create a temporary file to write out config data read from stdin.
+	tmpFile, err := ioutil.TempFile(dir, filePrefix)
+	if err != nil {
+		return configFilenames, err
+	}
+	// Store each line into the current config object.
+	var currentObject string
```

Comment:
I think using https://golang.org/pkg/strings/#Builder would be better here. I'm not really concerned about the performance of string concatenation, but I think using the Builder would be clearer.